img/c/cat.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 280 150"><path fill="#FFCD11" d="M57.5 154c6.3-6.2 12.9-12.8 20-18.7 11.1-9.3 22.6-18.2 34-27.3 7-5.7 14-11.5 21.1-16.9 1.1-.8 4.3-.3 5.5.7 11.3 9.3 22.3 19 33.5 28.3 11.3 9.5 22.8 18.8 34.1 28.2l6.2 5.4c-51.3.3-102.6.3-154.4.3z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 29.29 17.73"><path d="M8.39 14.8c.67-.56.78-1.25.78-1.87v-2.6H5.53v3.53c0 .51-.41.92-.91.92s-.88-.41-.88-.92V3.85c0-.51.38-.91.88-.91s.91.41.91.91v3.56h3.63V3.72C9.17 1.68 7.83 0 4.59 0 1.26 0 0 1.68 0 3.72v10.45c0 2.02 1.6 3.55 3.65 3.55h1.13zm14.33.41V3.73h-2.51V.34h9.08v3.39h-2.52V17.7h-1.08z"/><path d="m24.35 17.73-8.85-7.41-9.16 7.41z" fill="#ffc400"/><path d="m14.53 8.32.98-5 .97 5zm3.5-7.96h-5.07L10.01 13.6l5.5-4.44 5.73 4.8z"/></svg>
```
I would use this one, it's not often you see just triangle when their logo is present